src/io/flutter/template/DartToplevelTemplateContextType.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2017 The Chromium Authors. All rights reserved.
+ * Use of this source code is governed by a BSD-style license that can be
+ * found in the LICENSE file.
+ */
+package io.flutter.template;
+
+import com.intellij.psi.PsiComment;
+import com.intellij.psi.PsiElement;
+import com.intellij.psi.util.PsiTreeUtil;
+import com.jetbrains.lang.dart.ide.template.DartTemplateContextType;
+import com.jetbrains.lang.dart.psi.DartClassDefinition;
+import org.jetbrains.annotations.NotNull;
+
+public class DartToplevelTemplateContextType extends DartTemplateContextType {
+  public DartToplevelTemplateContextType() {
+    super("DART_TOPLEVEL", "Top-level", Generic.class);
```

Comment:
Possibly, but I didn't come up with anything simple. About all I can think of is looking for an import statement that references Flutter, but that might not be good enough. Only people that have the Flutter plugin installed will be affected. Given how hard these things are to find I think we're safe waiting for bugs to be filed.